src/main/java/com/minecolonies/sounds/FishermanSounds.java

```diff
@@ -0,0 +1,83 @@
+package com.minecolonies.sounds;
+
+import com.minecolonies.util.SoundUtils;
+import net.minecraft.util.SoundEvent;
+import net.minecraft.util.math.BlockPos;
+import net.minecraft.world.World;
+
+import java.util.Random;
+
+/**
+ * Class containing the fisherman sounds.
+ */
+public class FishermanSounds
+{
+    /**
+     * Random generator.
+     */
+    private static Random rand = new Random();
+
+    /**
+     * Number of different sounds in this class.
+     */
+    private static final int NUMBER_OF_SOUNDS = 2;
+
+    /**
+     * Chance to say a phrase.
+     */
+    private static final int PHRASE_CHANCE = 25;
+
+    /**
+     * Chance to play a basic sound.
+     */
+    private static final int BASIC_SOUND_CHANCE = 100;
+
+    /**
+     * Containing the female fisherman sounds.
+     */
+    public static class Female
+    {
+        public static SoundEvent generalPhrases;
+        public static SoundEvent noises;
```

Comment:
![CRITICAL](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-critical.png) Make this "public static noises" field final [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=squid%3AS1444)
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Make noises a static final constant or non-public and provide accessors if needed. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=squid%3AClassVariableVisibilityCheck)
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Document this public field. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=squid%3AUndocumentedApi)